src/DynamoCoreWpf/UI/GuidedTour/Guide.cs

```diff
@@ -84,6 +93,7 @@ private void SubscribeFlowEvents()
             GuideFlowEvents.GuidedTourNextStep += GuideFlowEvents_GuidedTourNextStep;
             GuideFlowEvents.GuidedTourPrevStep += GuideFlowEvents_GuidedTourPrevStep;
             GuideFlowEvents.UpdatePopupLocation += GuideFlowEvents_UpdatePopupLocation;
+            GuideFlowEvents.UpdateLibraryInteractions += GuideFlowEvents_UpdateLibraryInteractions;
```

Comment:
We need to unsubscribe it later

Reply:
@QilongTang yes, this is already happening in the  "private void UnsubscribeFlowEvents()" method.
https://github.com/DynamoDS/Dynamo/blob/b7b0b1a1f78822ac3773efaf6dca381c16e56e41/src/DynamoCoreWpf/UI/GuidedTour/Guide.cs#L77